conf/ocsci/ocp_47_bm_workaround.yaml

```diff
@@ -0,0 +1,2 @@
+DEPLOYMENT:
```

Comment:
is this is a file needed for a workaround? if so, please open a w/a issue

Reply:
yes for now it's a WA